### DISCUSSION TITLE
Feature request: Pre V20.4 discord UI option

### DISCUSSION THREAD

**Discussion by Schmiddy2point0** [stoatchat/discussions#556] [Feature Suggestions]
I'm not a huge fan of the modernized UI that's basically the same as discords. I much prefer the old, non bubbly or tab ui we had pre 2020. This is also the reason why I've used a version of discord from 2019 for about a year and I'm just switching over to revolt (which is way cooler so far). 
I think it'd be cool if we had a legacy UI option in settings for this.

**Comment by Nightcaat** [comment]
I’ve currently put a couple hours into making a Discord UI theme and I’d be happy to tackle old Discord afterwards :)
